src/udev/udevd.c

```diff
@@ -534,7 +534,7 @@ static int worker_spawn(Manager *manager, struct event *event) {
         if (r < 0)
                 return log_error_errno(r, "Worker: Failed to enable receiving of device: %m");
 
-        r = safe_fork("(worker)", FORK_DEATHSIG, &pid);
+        r = safe_fork(NULL, FORK_DEATHSIG, &pid);
```

Comment:
BTW, I've tried to set the worker's process name, but it does not work. At least `journald` does not show process name correctly. `udevd` does not have enough privilege to change process name?